### PR TITLE
S02-01: Background Carousel with Card Presentation

### PR DESCRIPTION
**Pull request by NEL45**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]

File: src/View/User/Includes/_welcome_background_slideshow.html.twig [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<div class="slider">
+        <div class="load">
+        </div>
+        <div class="content">
+            <div class="principal">
+                <h1>lorem ipsum</h1>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+                <div class="card border-0" style="width: 18rem;">
+                <div class="card-body">
+                    <h5 class="card-title">Lorem ipsum</h5>
```
Comment (LucyMerlier) [inline src/View/User/Includes/_welcome_background_slideshow.html.twig]: from h1 to h5 just like that?

File: src/View/User/Includes/_welcome_background_slideshow.html.twig [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+<div class="slider">
+        <div class="load">
+        </div>
+        <div class="content">
+            <div class="principal">
+                <h1>lorem ipsum</h1>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+                <div class="card border-0" style="width: 18rem;">
+                <div class="card-body">
```
Comment (LucyMerlier) [inline src/View/User/Includes/_welcome_background_slideshow.html.twig]: indentation

File: src/View/User/Includes/_welcome_background_slideshow.html.twig [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{# User/Includes/_welcome_background_slideshow.html.twig #}
+
+<div class="slider">
+        <div class="load">
+        </div>
+        <div class="content">
+            <div class="principal">
+                <h1>lorem ipsum</h1>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+                <div class="card border-0" style="width: 18rem;">
```
Comment (LucyMerlier) [inline src/View/User/Includes/_welcome_background_slideshow.html.twig]: NO STYLE IN HTML

File: src/View/User/welcome.html.twig [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{# welcome.html.twig #}
```
Comment (LucyMerlier) [inline src/View/User/welcome.html.twig]: eh, I think I know that

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments + do you really need that many images right now? + weighing more than 200KB

**Review by ChristianOlmedo — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**